pyproject.toml

```diff
@@ -104,6 +104,10 @@ disable_error_code = ["arg-type", "operator", "union-attr"]
 module = "data_store/validation/towns_fund/fund_specific_validation/fs_validate_r4"
 disable_error_code = ["arg-type", "assignment"]
 
+[[tool.mypy.overrides]]
```

Comment:
Cheeky mypy disabling, @albertkol won't be happy! But maybe he's not got to fixing those particular type errors yet as it looks like the R4 file disables the same mypy check.